rest-api/dao/physical_measurements_dao.py

```diff
@@ -219,8 +219,6 @@ def set_measurement_ids(physical_measurements):
 
   def insert_with_session(self, session, obj):
     is_amendment = False
-    if obj.logPosition is not None:
```

Comment:
Note: we had to do this because obj gets its logPosition field set in here, and if the insert fails due to a PK violation, this method will get called again... it shouldn't fail at that point.